File: python/62_n_mod_m.py
import sys

with open(sys.argv[1], 'r') as fh:
    for line in fh.readlines():
        line = line.strip()
        if line == '':
            continue

        (n, m) = [int(n) for n in line.split(',')]
        while n >= m:
            n -= m

        print n

File: perl/62_n_mod_m.pl
open my $fh, '<', $ARGV[0];

for my $line ( <$fh> ) {
	next if $line =~ /^\s*$/;

	(my $n, my $m) = split /,/, $line;
	while ( $n >= $m ) {
		$n -= $m;
	}

	print "$n\n";
}

close $fh;

exit 0;

